central/notifiers/microsoftsentinel/README.md

```diff
@@ -0,0 +1,30 @@
+# Microsoft Sentinel Integration
```

Comment:
Would it be worth adding a link to azure docs in here?

Reply:
Addressed in: https://github.com/stackrox/stackrox/pull/12637/commits/6b37a9bcd38c9107f5b94012285de180a919673b

More eng documentation will follow.